src/main/twirl/com/lightbend/lagom/docs/faq.scala.html

```diff
@@ -0,0 +1,49 @@
+@import com.lightbend.lagom.docs.LagomContext
+
+@(implicit context: LagomContext)
+
+@page("FAQ") {
+  <div>
+    <h1>Frequently Asked Questions</h1>
+    <blockquote>
+    Have a question about Lagom? We've sorted out a sample of frequently asked questions and provided answers below. If your question is not covered here, try these resources:
+    <ul>
+      <li><a href="//gitter.im/lagom/lagom">Lagom Gitter channel</a>
+      <li><a href="//groups.google.com/forum/#!forum/lagom-framework">lagom-framework mailing list</a> </li>
+      <li><a href="//stackoverflow.com/questions/tagged/lagom">Stackoverflow Lagom questions.</a></li>
+    </blockquote>
+
+
+      <h3>Is Lagom open source or commercial?</h3>  Lagom is completely open
+      source and has a vibrant and growing community. However, you do have the
+      option to purchase a <a
+      href="https://www.lightbend.com/platform/subscription">Lightbend
+      subscription</a> that provides technical support as well as license to use
+      Enterprise Suite, which includes a range of powerful Application
+      Management, Intelligent Monitoring, Enterprise Integration and Advanced
+      Tooling.     </p> 
+      <h3>Does Lagom require me to use Cassandra for
+      persistence?</h2>  
+      <p> No, Lagom supports using either Cassandra or
+      relational databases, including MySQL, PostegreSQL, Oracle and H2.
+      The Lagom development environment provides and uses Cassandra by default becacause it offers great scalability and replication features. 
+      See more in the <a href="https://www.lagomframework.com/documentation/1.3.x/java/PersistentEntity.html">Java documentation</a> or the <a
+      href="https://www.lagomframework.com/documentation/1.3.x/scala/PersistentEntity.html">Scala documentation</a>. </p>      
+
+        <h3>Why are event interfaces present in
```

Comment:
Yup, see the discussion with Tim previously. As we get more questions, we can organize them in categories. This is just a start. I think that I got this info from a discussion you had with someone about the hello world sample. Is that right, does it demonstrate this, or was it a different sample? I'll add a link when you tell me the best one to link to.